### ISSUE TITLE
Mainline Sonoff SC support?

### ISSUE THREAD

**Issue by Cabalist** [xoseperez/espurna#1271]
I was curious if the Sonoff SC will be supported by the main Espurna repo?   I am using an older version from https://github.com/xoseperez/sonoffsc and miss some of the nice features of the updated codebase.

**Comment by xoseperez** [comment]
It is a work in progress. Check the SonoffSC branch.

**Comment by Cabalist** [comment]
Thanks!  You are doing amazing work!

**Comment by peterhoeg** [comment]
I'm super keen to try this out!

**Comment by peterhoeg** [comment]
@mcspr, are you by chance doing any work on getting this integrated?

**Comment by mcspr** [comment]
No. I also did a double-check, we don't have anything related to AT commands handling atm.
Did you mean to make the port?

**Comment by peterhoeg** [comment]
I do not realistically have time to work on that. I just happen to have one of these devices sitting in a drawer and would like to start using it.

**Comment by mcspr** [comment]
Most of the code is already there. SC-specific support consists of comms.ino + serial link helper functions that talk through the AT commands, everything else is *supposed* to be a sensor i.e. commsLoop() -> BaseSensor::tick(), read-out functions -> BaseSensor::value(...) and etc.

I could look at it some more, but no promises it will be soon

**Comment by peterhoeg** [comment]
That would be awesome!
